core/rio/jsonld/src/test/resources/testcases/ndjsonld/mates.ndjsonld

```diff
@@ -0,0 +1,3 @@
+{  "@id" : "http://ndjsonld.com/Mate1",  "@type":["http://schema.org/Person"], "http://schema.org/givenName":[{"@value":"Mate1"}]}
```

Comment:
I'm enjoying how our running parser examples now all seem to be based on Australian slang :)